src/components/GraphiQL.js

```diff
@@ -199,6 +200,10 @@ export class GraphiQL extends React.Component {
 
   componentDidMount() {
     if (!this.state.schema) {
+      this.setState({
+        showSpinner: true
```

Comment:
We should only show a spinner when there is an user-produced query to the server underway, not for the automated fetching of the schema. GraphiQL is still usable without a schema, just typeahead and linters are not yet enabled, that's fine.

This implementation also has a race condition where we could issue a query before the schema returns. The schema would return and turn off the spinner, but the query itself hasn't returned yet.